Modules/+Imaging/+Thorlabs/uc480.m

```diff
@@ -0,0 +1,126 @@
+classdef uc480 < Modules.Imaging
+    % Connects with old-style Thorlabs cameras.
+    
+    properties
+        maxROI = [-1 1; -1 1];
+        
+        prefs = {'exposure'};
+        
+        cam = []
+        MemId = []
+    end
+    properties(GetObservable, SetObservable)
+        exposure =      Prefs.Double(NaN, 'units', 'ms', 'min', 0, 'max', inf, 'allow_nan', true, 'set', 'set_exposure');
+%         serial_number = Prefs.String('', 'readonly', true, 'allow_empty', true);
+    end
+    properties(GetObservable, SetObservable)
+        bitdepth = 0;
+        resolution = [120 120];                 % Pixels
+        ROI = [-1 1;-1 1];
+        continuous = false;
+    end
+    methods(Access=private)
+        function obj = uc480()
+            % Open camera connection
+            NET.addAssembly('C:\Program Files\Thorlabs\Scientific Imaging\DCx Camera Support\Develop\DotNet\uc480DotNet.dll');
```

Comment:
comment to make sure this is added in this loc?

Reply:
Installs by default withe ThorCam, I think, so I'm not so worried. If not installed, it will error in a traceable way.

Reply:
try-catch around it to make the error more user-friendly for someone who loads this and has no idea how command center works?

Reply:
Done.